sirepo/template/radia_util.py

```diff
@@ -172,6 +147,31 @@ def _radia_material(material_type, magnetization_magnitude, h_m_curve):
 )
 
 
+def apply_bevel(g_id, beam_dir, gap_dir, trans_dir, obj_ctr, obj_size, bevel):
+
+    b = numpy.array(beam_dir)
+    g = numpy.array(gap_dir)
+    x = numpy.array(trans_dir)
+    sz = numpy.array(obj_size)
+    ctr = numpy.array(obj_ctr)
+    e = int(bevel.edge)
+    half_size = sz / 2
+    corner = ctr + half_size * [-x + g + b, x + g + b, x - g + b, -x - g + b][e]
+    trans_offset = bevel.amountTrans * x * [1, -1, -1, 1][e]
```

Comment:
naming the various lists `[1, -1, -1, 1]` as constants would be good, especially if they could be looked up based on use, i.e. the lists would be part of a dict.